### PR TITLE
remove rounded borders

### LINKED ISSUE #1061: Don't round anything

### PR DESCRIPTION
**Pull request by na3d**
Closes #1061

### CODE REVIEW

**Comment by eatonphil** [comment]
Screenshots?

File: scss/components/buttons.scss [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 .btn {
     text-decoration: none;
-    border-radius: 2px;
+    border-radius: 0px;
```
Comment (mparke) [inline scss/components/buttons.scss]: can be `0;`

File: scss/components/tabs/Tabs.scss [hunk 2]
```diff
@@ -22,9 +22,9 @@
       background: $light-gray;
       border: 1px solid #bbb;
       border-bottom: 0;
-      border-radius: 2px 2px 0 0;
-      -moz-border-radius: 2px 2px 0 0;
-      -webkit-border-radius: 2px 2px 0 0;
+      border-radius: 0;
```
Comment (mparke) [inline scss/components/tabs/Tabs.scss]: if we're applying vendor prefixes, let's be consistent in applying them. consider creating a mixin, here's an example straight from the docs:

```
@mixin border-radius($radius) {
  -webkit-border-radius: $radius;
      -moz-border-radius: $radius;
        -ms-border-radius: $radius;
                border-radius: $radius;
}

.box { @include border-radius(0); }
```

thoughts?

Comment (eatonphil) [inline scss/components/tabs/Tabs.scss]: As discussed in meetspace, I think using a prefix generator would be a nicer solution than us doing it manually anywhere.

**Comment by eatonphil** [comment]
Screenshots?

File: scss/components/tabs/Tabs.scss (hunk 2 above)
Comment (mparke) [inline scss/components/tabs/Tabs.scss]: +1 to autoprefixer, I'd suggest we look at using postcss, but may be beyond the scope here. https://github.com/linode/manager/issues/1087

Fix obvious problems here, ignore vendor prefixes for now?

**Comment by na3d** [comment]
screen shot 2017-01-24 at 11 34 19 am

**Comment by caker** [comment]
Still rounded corners in that screenshot. Can you find them? :)

**Comment by eatonphil** [comment]
Ones I'm seeing:

* [ ] Checkboxes
* [ ] .input-group (Mass action button group thingy)

**Comment by na3d** [comment]
The checkbox is a browser default.  Are we creating our own custom checkboxes?

**Comment by eatonphil** [comment]
@na3d sure

**Comment by na3d** [comment]
Backlogged the checkboxes since it cascades into other things needing changed.

**Comment by na3d** [comment]
Forgot to post this:
screen shot 2017-01-24 at 3 45 33 pm

**Pull request merged**